jfrog-cli/artifactory/commands/config_test.go

```diff
@@ -6,21 +6,69 @@ import (
 	"testing"
 )
 
-func TestConfig(t *testing.T) {
+func TestBasicAuth(t *testing.T) {
 	inputDetails := config.ArtifactoryDetails{Url: "http://localhost:8080/artifactory",
```

Comment:
I suggest adding the following comment above this line:
// API key is no longer allowed to be configured without providing a username.
// This test is here to make sure that old configurations (with API key and no username) are still accepted.